src/view/popup/usePopup.ts

```diff
@@ -137,7 +89,10 @@ export const usePopup = () => {
             if (message.command.bytesTransferredChanged) {
                 const _bytes = message.command.bytesTransferredChanged;
                 setBytesTransferred(_bytes);
-                const _emissions = calculateCarbon(_bytes, selectedCountries);
+                const _emissions = calculateCarbon(
```

Comment:
why the underscore?

Reply:
Habits from dart to mark a variable as private;

- Previously there was const `[emissions, setEmissions] = useState()` so I used `_emissions` here to prevent confusion
- What are your thoughts?